interview-questions.md

```diff
@@ -8,43 +8,77 @@ Then, research the question to expand on your answer. Even if you feel you have
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
```

Comment:
For all the interview questions, you gave great information and explanations. Remember to always use interview questions to share how you have used that tool. Sharing how you used the tool shows the interviewer that you have went beyond just reading about the coding principles.